src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -24,21 +26,29 @@ internal static class DependentProjectsFinder
         /// <summary>
         /// A helper struct used for keying in <see cref="s_dependentProjectsCache"/>.
         /// </summary>
-        private struct DefinitionProject
+        private readonly struct DefinitionProject : IEquatable<DefinitionProject>
         {
-#pragma warning disable IDE0052 // Remove unread private members - DefinitionProject is used as a key for dictionaries.
-            private readonly ProjectId _sourceProjectId;
+            private readonly ProjectId? _sourceProjectId;
             private readonly string _assemblyName;
```

Comment:
📝 This can be null as well since this is a struct.

```suggestion
            private readonly string? _assemblyName;
```

Reply:
Done.